### DISCUSSION TITLE
How to change the color of commit author and line in graph view?

### DISCUSSION THREAD

**Discussion by DominickVale** [jesseduffield/lazygit#2307] [Q&A]
![image](https://user-images.githubusercontent.com/52491108/205309295-7ed96f28-83f8-4cca-b0e8-edf89e249cde.png)
It's unreadable

**Comment by mark2185** [comment]
For changing the author color, try adding this to your `config.yml`.

```
gui:
  authorColors:
    'DominickVale': 'yellow'
```

**Reply by DominickVale** [reply]
Isn't that going to work only for my name?
Also, what about the graph lines?

**Reply by mark2185** [reply]
> Isn't that going to work only for my name?

I'm guessing you could write `'*'` instead of your name.

>  Also, what about the graph lines?

Don't know, that's why I haven't replied to that subquestion :)

**Reply by DominickVale** [reply]
Alrighty, thank you very much anyway

**Reply by mark2185** [reply]
Have you tried changing [these](https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#default) colors?

**Reply by DominickVale** [reply]
yep, i tried setting them all in red but the graph didn't change at all
